commit_blobs.go

```diff
@@ -34,49 +36,76 @@ func (commitBlobsTable) String() string {
 	return printTable(CommitBlobsTableName, CommitBlobsSchema)
 }
 
-func (commitBlobsTable) Resolved() bool { return true }
-
 func (commitBlobsTable) Name() string { return CommitBlobsTableName }
 
 func (commitBlobsTable) Schema() sql.Schema { return CommitBlobsSchema }
 
-func (t *commitBlobsTable) TransformUp(f sql.TransformNodeFunc) (sql.Node, error) {
-	return f(t)
+func (t *commitBlobsTable) WithFilters(filters []sql.Expression) sql.Table {
+	nt := *t
+	nt.filters = filters
+	return &nt
 }
 
-func (t *commitBlobsTable) TransformExpressionsUp(f sql.TransformExprFunc) (sql.Node, error) {
-	return t, nil
+func (t *commitBlobsTable) WithProjection(colNames []string) sql.Table {
+	return t
```

Comment:
doesn't accept this table pushdowns of projections?

Reply:
Nope, only blobs and files, which don't read the blob if the blob_content is not pushed down